src/daemon/rpc_command_executor.cpp

```diff
@@ -91,7 +115,7 @@ namespace {
       << "height: " << boost::lexical_cast<std::string>(header.height) << std::endl
       << "depth: " << boost::lexical_cast<std::string>(header.depth) << std::endl
       << "hash: " << header.hash << std::endl
-      << "difficulty: " << header.wide_difficulty << std::endl
+      << "difficulty: " << diff_to_dec(header.wide_difficulty) << std::endl
```

Comment:
Boost's cpp_int supports construction from a hex string (https://www.boost.org/doc/libs/1_56_0/libs/multiprecision/doc/html/boost_multiprecision/tut/conversions.html), so we can just do
```c++
       << "difficulty: " << cryptonote::difficulty_type(header.wide_difficulty) << std::endl
```
instead of introducing the new function `diff_to_dec`.